### ISSUE TITLE
508 Compliance and Accessibility

### ISSUE THREAD

**Issue by YushYush** [YACS-RCOS/yacs#142]
508 Compliance and Accessibility Review using accessibility tool. Determine which areas of software meet 508 technical standards in each section. Initial testing on webpage and future testing on mobile apps.  Should meet graphic contrast standards, should be accessible to audio browsers and screen readers, should allow basic forms of user interaction including search and select. 

Potential testing with various users and browsers.

**Comment by copperwater** [comment]
Can you provide a link to these standards?
